controllers/countriesControllers.js

```diff
@@ -0,0 +1,77 @@
+const countriesRepositories = require('../repositories/countriesRepositories');
+const uuid = require('uuid'); //uuid needs to be installed
+const countriesUtilities = require('../utilities/countriesUtilities');
+
+
+//get all countries
+exports.getAllCountries = (req, res) => {
+    res.success(countriesRepositories.getAllCountries());
+}
+
+//get a single country
+exports.getCountry = (req, res) => {
+    const availableCountries = countriesRepositories.getCountry(req.params.name);
+    if (availableCountries.length !== 0) {
+        res.success(availableCountries);
+    }
+    else {
+        const countryName = countriesUtilities.capitalizeFirstLetter(req.params.name);
+        return res.notFound();
+    }           
+};
+
+// post/add a country
+exports.postCountry = (req, res) => {
+
+            const newCountry = {
+                id: id = uuid.v4(), 
+                name: req.body.name, 
+                capital: req.body.capital
+            };
+
+            if (!newCountry.name || !newCountry.capital) {
+                return res.customedError(`Please include a name AND a capital.`);
+            }
+            else {
```

Comment:
aici nu ai nevoie de else. Daca intra in if atunci se returneaza oricum si nu mai se executa tot ce ii dupa if, iar daca nu intra in if, atunci se executa tot ce ii dupa. (Daca nu ai avea return in if, atunci da, ar fi nevoie de else)

In acest fel mai scapi de un indentation